cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md

```diff
@@ -24,27 +24,27 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to learn about the nature of the CMS physics objects and the corresponding variables and terminology."
 
-- Go to ["CMS Physics Objects"](http://opendata.cern.ch/about/CMS-Physics-Objects).
+- Go to ["CMS Physics Objects"](http://opendata-dev.web.cern.ch/articles/cms-physics-objects-2011).
 
 ---
 
 #### Deciding which datasets to explore
 
-> "I want to find out whether I should go for data from 2010 or 2011 (both are pp data at 7 TeV)."
+> "I want to find out whether I should go for data from 2010 or 2011 (both are pp data at 7 TeV) or 2012 (pp data at 8 TeV)."
 
 - The 2010 data have been released first; have fewer, smaller datasets with better low-p<sub>T</sub> tracking, low trigger thresholds, low pile-up and more/simpler analysis/validation examples; but have no MC. If you do not need MC or maximal statistics, you might want to try 2010 data first (simpler).
-- The 2011 data have more statistics, more diverse datasets, many associated MC sets, and a slightly more advanced VM environment. If you are immediately interested in maximal statistics and/or MC acceptance corrections you should go for 2011 data (more sophisticated).
+- The 2011/2012 data have more statistics, more diverse datasets, many associated MC sets, and a slightly more advanced VM environment. If you are immediately interested in maximal statistics and/or MC acceptance corrections you should go for 2011/2012 data (more sophisticated).
 - Information on the respective luminosities and pile-up rates vs time can be found on [this Twiki page](https://twiki.cern.ch/twiki/bin/view/CMSPublic/LumiPublicResults#Multi_year_Collisions_Plots).
-- If you want to try both datasets, you will need to use the appropriate VMs.
+- If you want to try both datasets, you will need to use [the appropriate VMs](http://opendata-dev.web.cern.ch/search?page=1&size=20&tags=VM&experiment=CMS).
 
 <center>–––</center>
 
-> "I want to install the CMS software environment on the virtual machine (separately for 2010 and 2011 data) which is needed for access to and analysis of CMS Research level data."
+> "I want to install the CMS software environment on the virtual machine (separately for 2010 and 2011/2012 data) which is needed for access to and analysis of CMS Research level data."
 
-- Go to ["Start analysing the data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-open-data), and
-- choose "2011" (default) or "2010".
+- Install [the approriate VM](http://opendata-dev.web.cern.ch/search?page=1&size=20&tags=VM&experiment=CMS), 2010 VM for 2010 data, and 2011 VM for 2011/2012 data.
```

Comment:
Not strictly a review of this text, but should we re-name the second VM "2011/2012 VM"?